### PR TITLE
Add 7PHP Swag

### LINKED ISSUE #185: Add 7PHP swag
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : [7PHP](https://7php.com/)

Q. Where have you seen the announcement (URL)?  
A. : https://7php.com/stickers/

Q. What kind of swag (clothing, stickers, ...)?  
A. : stickers

Q. Have you received swag from this?  
A. : No

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Yes?

Note I don't think we should add it to the list as it's not really interesting, but it's still free swag..

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by vivekptnk**
## Wild swag opportunity appeared

- [x] I've checked that this isn't a duplicate pull request.
- [x] I've checked that this is linked to a new swag opportunity issue <!>

Implement swag opportunity: 7PHP

Closes #185

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -241,5 +241,14 @@
...
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "7PHP",
+    "difficulty": "easy",
+    "description": "If you are a fan of 7PHP, just send email at 'blogcontact@7php.com' and receive your free stickers ",
+    "reference": "https://7php.com/stickers/",
+    "image": "https://7php.com/wp-content/uploads/2018/05/7php-sticker-v2.jpg",
+    "dateAdded": "2018-12-17T21:20:05.000Z",
```
Comment (vikaspotluri123) [inline data.json]: This should be `2018-11-23T17:27:00.000Z`

File: data.json [hunk 2]
```diff
@@ -241,5 +241,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "7PHP",
+    "difficulty": "easy",
+    "description": "If you are a fan of 7PHP, just send email at 'blogcontact@7php.com' and receive your free stickers ",
```
Comment (vikaspotluri123) [inline data.json]: Could you add some ending punctuation to be consistent with everything else

File: data.json [hunk 3]
```diff
@@ -241,5 +241,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "7PHP",
```
Comment (vikaspotluri123) [inline data.json]: This file should be alphabetical sorted by name, please move this opportunity to the top ⬆

**Comment by vivekptnk** [comment]
resolved the issues, please check.

File: data.json [hunk 4]
```diff
@@ -1,4 +1,13 @@
 [
+   {
+    "name": "7PHP",
+    "difficulty": "easy",
+    "description": "If you are a fan of 7PHP, just send email at 'blogcontact@7php.com' and receive your free stickers.",
```
Comment (aslafy-z) [inline data.json]: I feel like the email shouldn't be in the description. It exposes it more to bots and it's easy to find anyway

Comment (vivekptnk) [reply data.json]: ill remove the email immediately.

File: data.json [hunk 5]
```diff
@@ -1,4 +1,13 @@
 [
+   {
+    "name": "7PHP",
+    "difficulty": "easy",
+    "description": "If you are a fan of 7PHP, just send email and receive your free stickers.",
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "description": "If you are a fan of 7PHP, just send an email and receive your free stickers.",
```rs.

File: data.json [hunk 6]
```diff
@@ -1,4 +1,13 @@
 [
+   {
+    "name": "7PHP",
+    "difficulty": "easy",
+    "description": "If you are a fan of 7PHP, just send an email and receive your free stickers.",
```
[the remark this answers is not in the document]
Comment (aslafy-z) [reply data.json]: Did it.

**Comment by aslafy-z** [comment]
Thanks @vivekptnk! It's live now at https://devswag.io!

**Pull request merged**